node/cn/handler.go

```diff
@@ -465,6 +466,14 @@ func (pm *ProtocolManager) handle(p Peer) error {
 }
 
 func (pm *ProtocolManager) processMsg(msgCh <-chan p2p.Msg, p Peer, addr common.Address, errCh chan<- error) {
+	defer func() {
```

Comment:
How about moving this recover logic into `pm.handleMsg()`?
And it can return panic error. Then present logic return `processMsg()`.
I think it is better flow for the panic case cased by a peer

Reply:
@KimKyungup It could be, but the defer function would be declared for every p2p message the node received.

Reply:
I got it. This is better. Good.